src/Symfony/Component/Form/Form.php

```diff
@@ -815,7 +815,7 @@ public function getErrors($deep = false, $flatten = true)
 
                 $iterator = $child->getErrors(true, $flatten);
 
-                if (0 === count($iterator)) {
+                if (!$iterator) {
```

Comment:
same, `$iterator` can be a `\Countable` (FormErrorIterator), this wouldn't be equivalent